src/app/@ansyn/menu-items/cases/components/cases-auto-save/cases-auto-save.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, HostListener, Input } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { UpdateCaseAction } from '@ansyn/menu-items/cases/actions/cases.actions';
+import { Case } from '@ansyn/core/models/case.model';
+
+@Component({
+	selector: 'ansyn-cases-auto-save',
+	templateUrl: './cases-auto-save.component.html',
+	styleUrls: ['./cases-auto-save.component.less']
+})
+export class CasesAutoSaveComponent {
+	@Input() currentCase: Case;
```

Comment:
input `caseId` and select case from store is more safeabily

Reply:
💪